pgml-cms/docs/api/sql-extension/pgml.train/README.md

```diff
@@ -37,7 +37,7 @@ pgml.train(
 | `task`          | `'regression'`                                        | The objective of the experiment: `regression`, `classification` or `cluster`                                                                                                                                                                                                                 |
 | `relation_name` | `'public.search_logs'`                                | The Postgres table or view where the training data is stored or defined.                                                                                                                                                                                                                     |
 | `y_column_name` | `'clicked'`                                           | The name of the label (aka "target" or "unknown") column in the training table.                                                                                                                                                                                                              |
-| `algorithm`     | `'xgboost'`                                           | <p>The algorithm to train on the dataset. |
+| `algorithm`     | `'xgboost'`                                           | <p>The algorithm to train on the dataset.</p> |
```

Comment:
We don't need the \<p\> if we're not nesting multiple links, but why remove the links?
```suggestion
| `algorithm`     | `'xgboost'`                                           | The algorithm to train on the dataset. |
```

Reply:
The links 404'ed. Do the pages still exist?

Reply:
Yeah, these are supposed to link to 
- Regression: https://postgresml.org/docs/api/sql-extension/pgml.train/regression
- Classification: https://postgresml.org/docs/api/sql-extension/pgml.train/classification